### PR TITLE
Page models in lexicon tests follow the Protractor style guide

### PR DESCRIPTION
**Pull request by MWiss**

### CODE REVIEW

File: test/app/languageforge/lexicon/pages/editorPage.js [hunk 1]
```diff
@@ -65,12 +67,10 @@ function EditorPage() {
 
     // Search typeahead
     search: {
-      input: this.browseDiv.element(by.css('div.typeahead')).element(by.css('input')),
-      clearBtn: this.browseDiv.element(by.css('div.typeahead')).element(by.className('fa-times')),
-      results: this.browseDiv.element(by.css('div.typeahead'))
-        .all(by.repeater('e in typeahead.searchResults')),
-      matchCountElem: this.browseDiv.element(by.css('div.typeahead'))
-        .element(by.binding('typeahead.matchCountCaption')),
+      input: this.browseDivSearch.element(by.css('input')),
```
Comment (megahirt) [inline test/app/languageforge/lexicon/pages/editorPage.js]: this is good.  well done.

File: test/app/languageforge/lexicon/pages/viewSettingsPage.js [hunk 2]
```diff
@@ -37,27 +34,14 @@ function ViewSettingsPage() {
       }.bind(this)
     },
     manager: {
+      // Might want better way to access this button, maybe Chris will know
```
Comment (megahirt) [inline test/app/languageforge/lexicon/pages/viewSettingsPage.js]: we should avoid putting names in the comments like this.  Instead something like:
`// unsure if there is a better way to access this button - Mark W 2018-01-11`

File: test/app/languageforge/lexicon/pages/viewSettingsPage.js [hunk 3]
```diff
@@ -37,27 +34,14 @@ function ViewSettingsPage() {
...
-
-  //noinspection JSUnusedGlobalSymbols
-  this.managerTab = {
-  };
-
-  this.showAllFieldsBtn = element(by.buttonText('Show All Fields'));
-
-  //noinspection JSUnusedGlobalSymbols
-  this.showCommonFieldsBtn = element(by.buttonText('Show Only Common Fields'));
-
```
Comment (megahirt) [inline test/app/languageforge/lexicon/pages/viewSettingsPage.js]: My assumption is that these properties are not used anywhere, and that's why you are removing them?

File: test/app/languageforge/lexicon/pages/viewSettingsPage.js [hunk 4]
```diff
@@ -6,20 +6,17 @@ function ViewSettingsPage() {
...
+  this.settingsMenuLink = element(by.id('settingsDropdownButton'));
+  this.viewSettingsLink = element(by.id('dropdown-view-settings'));
   this.get = function get() {
     browser.wait(expectedCondition.visibilityOf(this.settingsMenuLink), CONDITION_TIMEOUT);
     this.settingsMenuLink.click();
     browser.wait(expectedCondition.visibilityOf(this.viewSettingsLink), CONDITION_TIMEOUT);
     this.viewSettingsLink.click();
   };
 
-  //noinspection JSUnusedGlobalSymbols
```
Comment (megahirt) [inline test/app/languageforge/lexicon/pages/viewSettingsPage.js]: I assume this is not used anywhere and that's why you're removing it?

File: test/app/languageforge/lexicon/pages/viewSettingsPage.js (hunk 3 above)
Comment (MWiss) [reply test/app/languageforge/lexicon/pages/viewSettingsPage.js]: Yes, @megahirt had mentioned just removing unused properties could be removed. After confirming this property wasn't used anywhere, I removed it.

File: test/app/languageforge/lexicon/pages/viewSettingsPage.js (hunk 4 above)
Comment (MWiss) [reply test/app/languageforge/lexicon/pages/viewSettingsPage.js]: Yes, after making sure no tests used them, I removed certain items tagged with "//noinspection JSUnusedGlobalSymbols" if I found them distracting.

File: test/app/languageforge/lexicon/pages/viewSettingsPage.js (hunk 2 above)
Comment (MWiss) [reply test/app/languageforge/lexicon/pages/viewSettingsPage.js]: Updated the comment, will keep this in mind for the future.

**Pull request merged**